### ISSUE TITLE
Feature request: swap Joystick ports

### ISSUE THREAD

**Issue by smileybob** [MiSTer-devel/C64_MiSTer#96]
It would be good if we could swap joystick ports by using a shortcut instead of having to go back into the menu system. It would be also good if that shortcut could be added to a joystick button, this would make swapping ports so much easier.
Thank you and great work on the core 👍

**Comment by sorgelig** [comment]
Shortcuts are easy to forget while OSD option shows the state.
On real C64 you have to physically re-plug the joystick, so i don't think any game relay on quick and often swap. OSD option i think is adequate reflection of HW config.

**Comment by smileybob** [comment]
Not really, on a real c64 people leave 2 joysticks plugged in to evade having to swap ports which is a pain or there are devices you can buy which switches ports on a real c64 (it’s obviously a problem). There’s already some shortcuts for sound and auto fire, if we could have the swap ports assigned to a button press it would be very much appreciated and helpful 😊
